ai/GameDomainObjects/field.py

```diff
@@ -31,12 +31,26 @@ def __init__(self, line: Dict):
         self.thickness = line["thickness"]
 
 
+def convert_field_circular_arc(field_arcs: Dict):
+    result = {}
+    for arc in field_arcs:
+        result[arc["name"]] = FieldCircularArc(arc)
```

Comment:
name represent what?

Reply:
The protobuf arcs list looks like this:
```
[
{
  "name": "CentralCircle",
  "degree": 20,
  "radius": 100,
  ...
},
...
]
````

I want them like this:
```
{"CentralCircle" :
 {
  "degree": 20,
  "radius": 100,
  ...
  },
...
}
```

Reply:
make sense